tests/e2e/support/api/davSpaces/spaces.ts

```diff
@@ -4,6 +4,22 @@ import join from 'join-path'
 import { getPersonalSpaceId } from '../graph'
 import { config } from '../../../config'
 
+export const folderAlreadyExists = async ({
```

Comment:
`already` have no effect :smile:
```suggestion
export const folderExists = async ({
```

Reply:
sure !!